mmv1/products/dataplex/Datascan.yaml

```diff
@@ -0,0 +1,640 @@
+               name: 'uniquenessExpectation'
+               description: |
+                 ColumnAggregate rule which evaluates whether the column has duplicates.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'statisticRangeExpectation'
+               description: |
+                 ColumnAggregate rule which evaluates whether the column aggregate statistic lies between a specified range.
+               properties:
+                 - !ruby/object:Api::Type::Enum
+                   name: 'statistic'
+                   values:
+                     - :STATISTIC_UNDEFINED
+                     - :MEAN
+                     - :MIN
+                     - :MAX
+                 - !ruby/object:Api::Type::String
+                   name: 'minValue'
+                   description: |
+                     The minimum column statistic value allowed for a row to pass this validation.
+                     At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::String
+                   name: maxValue
+                   description: |
+                     The maximum column statistic value allowed for a row to pass this validation.
+                     At least one of minValue and maxValue need to be provided.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMinEnabled'
+                   default_value: false
+                   description: |
+                     Whether column statistic needs to be strictly greater than ('>') the minimum, or if equality is allowed.
+                     Only relevant if a minValue has been defined. Default = false.
+                 - !ruby/object:Api::Type::Boolean
+                   name: 'strictMaxEnabled'
+                   default_value: false
+                   description: |
+                     Whether column statistic needs to be strictly lesser than ('<') the maximum, or if equality is allowed.
+                     Only relevant if a maxValue has been defined. Default = false.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'rowConditionExpectation'
+               description: |
+                 Table rule which evaluates whether each row passes the specified condition.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'sqlExpression'
+                   description: |
+                     The SQL expression.
+             - !ruby/object:Api::Type::NestedObject
+               name: 'tableConditionExpectation'
+               description: |
+                 Table rule which evaluates whether the provided expression is true.
+               properties:
+                 - !ruby/object:Api::Type::String
+                   name: 'sqlExpression'
+                   description: |
+                     The SQL expression.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'dataProfileSpec'
```

Comment:
the [documentation](https://cloud.google.com/dataplex/docs/reference/rest/v1/DataProfileSpec) around this nested object is blank, are you aware of what this nested object should look like?

Reply:
The dataprofilespec is currently an empty message, but will actually get some defined fields in 1-2 weeks. We plan to create a fresh PR in case we wrap up this one by then.